Generals/Code/GameEngine/Source/Common/System/StackDump.cpp

```diff
@@ -86,6 +87,12 @@ _asm
  mov eax, ebp
  mov dword ptr [myebp] , eax
 }
+#else
```

Comment:
`#elif _WIN32`

Reply:
I would say basically whole file is only valid for ` _WIN32`. Code here is very windows specific... To target other platforms/OSes, big parts of platform specific code would need to be written for scratch (for given platform).

Also code is currently for x86 (32-bit) only. It would need more fixes to support windows x86_64. Currently it would not even compile (but that is out of scope of this PR).